package/src/renderer/Canvas.tsx

```diff
@@ -37,6 +37,9 @@ export const Canvas = forwardRef<SkiaDomView, CanvasProps>(
     const redraw = useCallback(() => {
       innerRef.current?.redraw();
     }, [innerRef]);
+    const getNativeId = useCallback(() => {
+      return innerRef.current?.nativeId;
```

Comment:
here we may want to do `innerRef.current?.nativeId ?? 0` so typescript marks the return type as number.
Or throw an error if the nativeId cannot be found